tests/causal_refuters/test_data_subset_refuter.py

```diff
@@ -11,6 +11,7 @@ def test_refutation_data_subset_refuter_continuous(self, error_tolerance, estima
         refuter_tester = TestRefuter(error_tolerance, estimator_method, "data_subset_refuter")
         refuter_tester.continuous_treatment_testsuite()  # Run both
 
+    @pytest.mark.advanced

```

Comment:
Most of the files use "mark.advanced" rather than pytest.mark.advanced.  Is this contextual or would it be better to be consistent?  or am I being too pedantic :)

Reply:
I've found both in the codebase - basically which one you use depends on whether you have `import pytest` or `from pytest import mark` in your file. I'm fine standardizing on using `@mark.advanced` though.

Reply:
Ok, let's standardize on mark.advanced then.